easybuild/easyconfigs/s/snap/snap-2013-11-29-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,40 @@
+#
+# Author: Fenglai Liu
+# fenglai@accre.vanderbilt.edu
+# Vanderbilt University
+#
+easyblock = 'MakeCp'
+
+name = 'snap'
+version = '2013-11-29'
+
+homepage = 'http://korflab.ucdavis.edu/software.html'
+description = """SNAP is a general purpose gene finding program suitable for both eukaryotic and prokaryotic genomes. 
+SNAP is an acroynm for Semi-HMM-based Nucleic Acid Parser."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+
+source_urls = ['http://korflab.ucdavis.edu/Software']
+sources = [SOURCE_TAR_GZ]
+patches = ['snap_makefile_correction.patch']
```

Comment:
@murfreesboro Please include the version in the name of the patch, for example `SNAP-%(version)s_makefile-correction.patch`

Reply:
added the version number in the patch.